content/blog/understanding-hpe-greenlake-identity-access-management.md

```diff
@@ -0,0 +1,291 @@
+| HPCaaS Viewer                                   | View only                                                                                                                                                            |     |
+
+For more detail on Roles, please refer to the HPE GreenLake Central User Guide which is available here: [HPE GreenLake Central User Guide](https://support.hpe.com/hpesc/public/docDisplay?docId=a00092451en_us&page=index.html)
+
+* Owner Roles apply to different aspects of Services. Some Administrator Roles are only available to HPE ITOC and DevOps teams.
+* Contributor Roles also apply to Services. These Roles allow non-admin operations within a Service.
+* Viewer Roles are 'read-only' roles which can be used to allow Users to access services but not to modify them in any way.
+* Custom Roles can also be defined. These allow arbitrary collections of Permissions to be combined into one or more Roles which can be assigned as normal.
+
+### Tenant On-boarding
+
+New tenants are requested via a HPE Representative. The request includes an initial contact email address. This new user account is associated with the new Tenant. Once the Tenant is created and the billing account is promoted into Production, the contact email address receives an email invitation to activate their account. The new user can then log into GreenLake Central and can start the process of setting up their Spaces, User Groups, Role Assignments, etc. They can also invite other users to join the Tenant and perform other activities relevant to the specific user permissions.
+
+### Inviting Users
+
+Once a Tenant Administrator is on-boarded, they can invite their users to join the Tenant.
+
+1. Access the Identity & Access Service within the Wrench icon.
+2. Select the Users Tab
+3. Under the Actions pull-down, choose 'Invite User'
+
+![](/img/invite-user.png)
+
+The invited User receives an email inviting them to join the Tenant. Once the User activates their account they can log into GreenLake Central and switch to the Tenant. The Tenant Administrator can add the new User to various User Groups, etc.
+
+### Creating and modifying User Groups
+
+1. Access the Identity & Access Service within the Wrench icon.
+2. Select the User Groups Tab
+3. To modify an existing User Group, click on it from the list of User Groups
+4. 1. Members can be added by selecting the Members Tab and selecting 'Add Members' under the Actions pull-down
+   2. Members can be removed from the User Group by clicking on the Trash icon beside the Member name
+5. To create a new User Group, click on the 'Create User Group' button
+6. 1. Enter a User Group Name and Description
+   2. Once the User Group has been created it may be modified using the instructions above
+
+### Creating and modifying Spaces
+
```

Comment:
The number list and sub list is not correctly numbered.

Reply:
As above